exchanges/binance/binance_wrapper.go

```diff
@@ -714,16 +714,22 @@ func (b *Binance) UpdateAccountInfo(ctx context.Context, assetType asset.Item) (
 		acc.Currencies = currencyDetails
 
 	case asset.USDTMarginedFutures:
-		accData, err := b.UAccountBalanceV2(ctx)
+		accData, err := b.UAccountInformationV2(ctx)
 		if err != nil {
 			return info, err
 		}
 		var currencyDetails []account.Balance
-		for i := range accData {
+		for i := range accData.Assets {
+			currencyDetails = append(currencyDetails, account.Balance{
+				CurrencyName: currency.NewCode(accData.Assets[i].Asset),
+				TotalValue:   accData.Assets[i].WalletBalance,
+				Hold:         accData.Assets[i].WalletBalance - accData.Assets[i].AvailableBalance,
+			})
+		}
+		for i := range accData.Positions {
```

Comment:
This is your open order positions and not really what this function is for which is your wallet balances. Technically it is part of your account, but this seems more suited to the `GetActiveOrders` function. It doesn't fully fit either as you are converting currency pairs into currency codes

Reply:
yeah, i agree that it's a poor fit here